ngraph/core/reference/include/ngraph/runtime/reference/interpolate.hpp

```diff
@@ -226,6 +222,16 @@ namespace ngraph
                     , m_out_shape{out_shape}
                     , m_scales{scales}
                 {
+                    size_t input_rank = input_data_shape.size();
+                    std::vector<float> all_scales(input_rank, 1.0f);
+                    size_t num_of_axes = axes.size();
+
+                    for (size_t i = 0; i < num_of_axes; ++i)
+                    {
+                        all_scales[axes[i]] = scales[i];
```

Comment:
Please add the check that `scales` vector has the same size as `axes`.

Reply:
Done.